subgraphs/compound-forks/protocols/moonwell/src/mapping.ts

```diff
@@ -193,25 +212,97 @@ export function handleNewReserveFactor(event: NewReserveFactor): void {
 export function handleMint(event: Mint): void {
   let minter = event.params.minter;
   let mintAmount = event.params.mintAmount;
-  _handleMint(comptrollerAddr, minter, mintAmount, event);
+  let contract = CToken.bind(event.address);
+  let balanceOfUnderlyingResult = contract.try_balanceOfUnderlying(
+    event.params.minter
+  );
+  let balanceOfUnderlying = BigInt.fromI32(-1);
```

Comment:
Should make `BigInt.fromI32(-1)` a named constant

Reply:
cool let me name it `REVERTED` :)